sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -128,6 +146,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_ptl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-ptl-rt711.tplg",
 	},
+	{
+		.link_mask = BIT(1),
+		.links = ptl_rt722_l1,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-ptl-rt722.tplg",
+	},
```

Comment:
nitpick: Do you mind to reorder it with link number? I.e. Move it to after the "ptl_rt722_only" item so that the rt722 link adrs will be link 0, 1, 3 order.

Reply:
Done